src/plugins/discover/public/components/discover_grid_flyout/discover_grid_flyout.tsx

```diff
@@ -251,7 +267,18 @@ export function DiscoverGridFlyout({
           )}
         </EuiFlyoutHeader>
         <EuiFlyoutBody>{bodyContent}</EuiFlyoutBody>
```

Comment:
@lukasolson this comment it referring to the element being rendered in the `EuiFlyoutBody` 

let's bring this over the finishing line by adjusting the responsiveness of the action column of the . something like this should work with a change at the beginning of the  `DocViewerTable` component

https://github.com/elastic/kibana/blob/f7341700b9cbed2fd6c2fa623d189e214edf4aa4/src/plugins/unified_doc_viewer/public/components/doc_viewer_table/table.tsx#L111-L125

```TypeScript
  const { euiTheme } = useEuiTheme();
  const [ref, setRef] = useState<HTMLDivElement | HTMLSpanElement | null>(null);
  const dimensions = useResizeObserver(ref);
  const showActionsInsideTableCell = dimensions?.width
    ? dimensions.width > euiTheme.breakpoint.m
    : false;
```
    
 With  `setRef` being used the the `EuiFlexGroup` wrapper
 
 Then the action column would no longer be responsive to the window width, but to the flyout width, which is a better pattern anyway
 
 Thx for the feedback @andreadelrio and `useResizeObserver` hint @davismcphee

Reply:
One other note: we should also make the single document and surrounding documents buttons responsive based on the flyout width since they currently use the window width too. We should be able to use the same resize observer approach for both the buttons and table actions.

>Should we use `euiTheme.breakpoint.m` or `euiTheme.breakpoint.l`?

We could use a preset breakpoint, but it might even be better to just use `euiTheme.base * X` and get a value close to when the UI elements start to "break".